src/main/webapp/itempagescript.js

```diff
@@ -143,16 +147,24 @@ function getItemPageComments(comments) {
  *
  * @param { string } comment - the comment including the username, message, and timestamp
  * @param { boolean } belongsToUser - boolean that shows whether or not the current user has posted a comment
+ * @param { string } email - the email of the user
  * @returns { html element }  returns a list element
  */
 function createListElement(comment, belongsToUser, email) {
   const liElement = $('<li class="list-group-item"></li>');
   liElement.text(comment);
   if (belongsToUser) {
     liElement.append($('<i class="fa fa-trash" style="float:right;"></i>'));
-    liElement.append($('<img src="'+ "https://icotar.com/avatar/"+ email.charAt(0) +".png?s=23" + '"style=float:left;"></i>'));
   }
-
+  liElement.append(
+    $(
+      '<img class="pr-1" src="' +
+        "https://icotar.com/avatar/" +
+        email.charAt(0) +
```

Comment:
You can use the helper function that returns the avatar url, so we don't have duplicated logic.